buildSrc/build.gradle.kts

```diff
@@ -0,0 +1,9 @@
+import org.gradle.kotlin.dsl.`kotlin-dsl`
+
+plugins {
+    `kotlin-dsl`
+}
+
+repositories {
+    mavenCentral()
+}
```

Comment:
It's better to avoid using the `buildSrc` directory because when you do a change in your grade files all the cache will be cleared.
There is something called `convention plugins` that you can use along with the gradle version catalog.
Check my project, you see that I. use a directory called `build-logic`.